src/features/collectables/NftDetailsScreen.tsx

```diff
@@ -0,0 +1,155 @@
+
+LogBox.ignoreLogs([
+  'Non-serializable values were found in the navigation state',
+])
+
+type Route = RouteProp<CollectableStackParamList, 'NftDetailsScreen'>
+
+const NftDetailsScreen = () => {
+  const route = useRoute<Route>()
+  const navigation = useNavigation<CollectableNavigationProp>()
+  const COLLECTABLE_HEIGHT = ww
+  const safeEdges = useMemo(() => ['bottom'] as Edge[], [])
+  const backEdges = useMemo(() => ['top'] as Edge[], [])
+
+  const { t } = useTranslation()
+
+  const { collectable } = route.params
+
+  const spacing = useSpacing()
+
+  const handleSend = useCallback(() => {
+    navigation.navigate('TransferCollectableScreen', {
+      collectable,
+    })
+  }, [collectable, navigation])
+
+  const handleInfoPress = useCallback(() => {
+    if (collectable.json) {
+      navigation.push('NftMetadataScreen', {
+        metadata: collectable.json,
+      })
+    }
+  }, [collectable.json, navigation])
+
+  const backgroundImageUri = useMemo(() => {
+    return collectable?.json?.image
+  }, [collectable.json])
+
+  if (!collectable.json || !backgroundImageUri) {
+    return null
+  }
+
+  return (
+    <BackScreen
+      padding="none"
+      title={t('collectablesScreen.nfts.nftDetialTitle')}
+      backgroundImageUri={backgroundImageUri}
+      edges={backEdges}
+      TrailingIcon={InfoIcon}
+      onTrailingIconPress={handleInfoPress}
+    >
+      <Animated.View entering={DelayedFadeIn} style={globalStyles.container}>
+        <ScrollView>
+          <SafeAreaBox
+            edges={safeEdges}
+            backgroundColor="transparent"
+            flex={1}
+            padding="m"
+            alignItems="center"
+          >
+            {collectable.json && (
+              <Box
+                shadowColor="black"
+                shadowOpacity={0.4}
+                shadowOffset={{ width: 0, height: 10 }}
+                shadowRadius={10}
+                elevation={12}
+              >
+                <ImageBox
+                  marginTop="l"
+                  backgroundColor="black"
+                  height={COLLECTABLE_HEIGHT - spacing.xl * 2}
+                  width={COLLECTABLE_HEIGHT - spacing.xl * 2}
+                  source={{ uri: collectable.json.image, cache: 'force-cache' }}
+                  borderRadius="xxl"
+                />
+              </Box>
+            )}
+            <Text
+              marginTop="l"
+              marginBottom="s"
+              marginHorizontal="l"
+              textAlign="center"
+              variant="h2"
+            >
+              {collectable.json.name}
+            </Text>
+            <Text variant="body2" color="grey600" marginBottom="xl">
+              {collectable.json.description || t('collectables.noDescription')}
+            </Text>
+            <Box
+              flexDirection="row"
+              marginBottom="xl"
+              marginTop="m"
+              marginHorizontal="xl"
+            >
+              <TouchableOpacityBox
```

Comment:
What does it mean to trash an NFT? Do we want to remove this for now?

Reply:
I think it means to burn an NFT. I asked Pete this question on the Figma. I rather not remove and just disable for now